### PR TITLE
Add stubs for pyflakes

### PR DESCRIPTION
**Pull request by AlexWaygood**

### CODE REVIEW

**Review by srittau — CHANGES_REQUESTED** [review]
A few small nits below, looks good apart from that, although I didn't check in-depth.

Also, should we really include `pyflakes.scripts`? It seems to me that this isn't intended to be imported and consists of re-imports only.

File: stubs/pyflakes/pyflakes/api.pyi [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+__all__ = ["check", "checkPath", "checkRecursive", "iterSourceCode", "main"]
+
+PYTHON_SHEBANG_REGEX: Pattern[bytes]
+
+def check(codeString: str, filename: str, reporter: Reporter | None = ...) -> int: ...
+def checkPath(filename, reporter: Reporter | None = ...) -> int: ...
+def isPythonFile(filename) -> bool: ...
+def iterSourceCode(paths: Sequence[Any]) -> Iterator[Any]: ...
+def checkRecursive(paths: Sequence[Any], reporter: Reporter) -> int: ...
```
[the remark this answers is not in the document]
Comment (AlexWaygood) [reply stubs/pyflakes/pyflakes/api.pyi:14]:
```suggestion
def checkRecursive(paths: Iterable[Any], reporter: Reporter) -> int: ...
```

File: stubs/pyflakes/pyflakes/api.pyi [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from collections.abc import Iterator, Sequence
```
Comment (AlexWaygood) [inline stubs/pyflakes/pyflakes/api.pyi:1]:
```suggestion
from collections.abc import Iterable, Iterator, Sequence
```

**Review by srittau — APPROVED** [review]

**Comment by AlexWaygood** [comment]
Thanks @srittau!

**Pull request merged**